includes/discovery/sensors/state/serverscheck.inc.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * serverscheck.inc.php
+ *
+ * LibreNMS state discover module for serverscheck flooding sensor
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2018 Marcus Pink
+ * @author     Marcus Pink <mpink@avantgarde-labs.de>
+ */
+
+
+$serverscheck_oids = array(
+    'sensor1Value.0' => '.1.3.6.1.4.1.17095.3.2.0',
+    'sensor2Value.0' => '.1.3.6.1.4.1.17095.3.6.0',
+    'sensor3Value.0' => '.1.3.6.1.4.1.17095.3.10.0',
+    'sensor4Value.0' => '.1.3.6.1.4.1.17095.3.14.0',
+    'sensor5Value.0' => '.1.3.6.1.4.1.17095.3.18.0',
+);
+
+$temp_x = 1;
+foreach ($pre_cache['serverscheck_control'] as $oid_name => $oid_value) {
+    if (str_contains($oid_name, 'name')) {
+        $tmp_oid = 'sensor' . $temp_x . 'Value.0';
+        $current = $pre_cache['serverscheck_control'][$tmp_oid];
+        if (str_contains($oid_value, array('Flooding', 'Leckage'))) {
```

Comment:
May as well move this into the first `if (str_contains($oid_name, 'name')) {` line and do a `&&`,